### ISSUE TITLE
Quiz questions in shiny

### ISSUE THREAD

**Issue by johnbde** [rstudio/learnr#477]
I am trying to embed quiz questions from learnr package in shiny app. Since v0.10.0 leanr quiz questions are mini shiny applications, i thought it would be easy to add it in shiny. Unfortunately it doesn't work (dont load quiz).

`
library(shiny)
library(learnr)

app <- shinyApp(
  
ui <- tagList(
    fluidPage(theme = shinytheme("cosmo")),
    navbarPage(
      "appTitle",
      tabPanel("Tutorial", 
               quiz(
                 question("Quiz question",
                          answer("1"),
                          answer("2"),
                          answer("3", correct = TRUE),
                          answer("4"),
                          allow_retry = TRUE
                 )
               )
      )
      )),
  
server <- function(input, output, session) {}

)

runApp(app)
`

**Comment by schloerke** [comment]
Correct. 😞 Unfortunately, `learnr` questions and exercises (#451) can _currently_ only be put into `learnr` tutorial R Markdown documents.

I hope we will eventually address #451 and be able to put an exercise and quiz questions in non-`learnr` tutorials documents.

**Comment by johnbde** [comment]
Thank you. Would be really helpful. Currently I am integrating it in shiny via iframe. Is there any easy or efficiency way?
